components/YearSelect.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  {{ regatta?.start_date.slice(0, 4) }}
+  <!-- Deze functie doet op het moment nog niks maar het is de bedoeling dat je-->
+  <!-- door deze functie uiteindelijk kan wisselen van jaartal - Duke 01/03/23 -->
+</template>
+
+<script setup lang="ts">
+import { Regatta } from "~/server/types";
+defineProps<{
+  regatta: Regatta;
```

Comment:
Deze type dus veranderen naar:
```suggestion
  regatta: Regatta | null;
```